setup.cfg

```diff
@@ -1,19 +1,20 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, W504
 exclude =
   bin.src,
-  doc,
+  doc/*,
   **/*/__init__.py,
   **/*/version.py,
   tests/.tests
 
 [tool:pytest]
 addopts = --flake8
 flake8-ignore = 
-  *.py E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W504
+  *.py E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503 W504
   bin.src/*.py ALL
+  doc/conf.py ALL
```

Comment:
Please consider allowing flake8 to run on this file. Append `  # noqa` to each intersphinx mapping line to eliminate the complaints about those. Everything else should be compliant.

Reply:
Done.